x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -50,9 +50,9 @@ statement
         )*
         ')')?
         statement                                                                                         #debug
-    | SHOW TABLES (tableLike=likePattern | tableIdent=tableIdentifier)?                                   #showTables
-    | SHOW COLUMNS (FROM | IN) (tableLike=likePattern | tableIdent=tableIdentifier)                       #showColumns
-    | (DESCRIBE | DESC) (tableLike=likePattern | tableIdent=tableIdentifier)                              #showColumns
+    | SHOW TABLES (INCLUDE FROZEN)? (tableLike=likePattern | tableIdent=tableIdentifier)?                 #showTables
+    | SHOW COLUMNS (INCLUDE FROZEN)? (FROM | IN) (tableLike=likePattern | tableIdent=tableIdentifier)     #showColumns
+    | (DESCRIBE | DESC) (INCLUDE FROZEN)? (tableLike=likePattern | tableIdent=tableIdentifier)            #showColumns
     | SHOW FUNCTIONS (likePattern)?                                                                       #showFunctions
     | SHOW SCHEMAS                                                                                        #showSchemas
     | SYS TABLES (CATALOG clusterLike=likePattern)?
```

Comment:
Notice the `SYS` commands do not require `INCLUDE FROZEN` since these are meant only for drivers in which case the option is set through the configuration.